### PR TITLE
Top toolbar

### PR DESCRIPTION
**Pull request by cchan987**
Creates a top toolbar with:
-Ability to open new file
-Reset the model layout
-undo button (inactive)
-redo button (inactive)
-Config options menu (active, needs to be populated)

### CODE REVIEW

**Comment by tcstewar** [comment]
Cool!  If you could take a stab at rebasing this to master, I'll start going through the code....

**Comment by studywolf** [comment]
might be worth getting the undo PR in first, so that the inactive buttons can be hooked up?

**Comment by tcstewar** [comment]
> might be worth getting the undo PR in first, so that the inactive buttons can be hooked up?

That is a good point.  I'll push on the undo people more....

File: nengo_viz/components/toolbar.py [hunk 1]
```diff
@@ -0,0 +1 @@
+##toolbar
```
Comment (tcstewar) [inline nengo_viz/components/toolbar.py]: Should this file exist?

**Comment by studywolf** [comment]
OK, rebased to master. Fixed things I broke during rebasing. Then I rebased to takehtmloutofjs and took the html out of viz_top_toolbar and put it into page.html.

I believe that everything works again, the config window is just empty, and the undo/redo have a TODO in viz_top_toolbar for where they should be hooked up.

**Comment by studywolf** [comment]
Oh one thing that came up. In viz_sim_control.css, to make things look right on my desktop I deleted the margin-top: 40px inside the .metrics-container.table. But now it's up too high on my laptop. Not sure what's going on there. That's actually a thing from takehtmloutofjs.

**Pull request merged**